From 280fdfd99933c76345cda1b7ea360094c771c446 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Sat, 3 Sep 2011 00:08:31 +0800
Subject: [PATCH 156/256] ENGR00155908 ipu: Fix linux-test build failure due
 to ipu lib change.

Fix linux-test build failure due to ipu lib change.
Need to copy new added ipu.h to rootfs.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 ipu/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ipu/Makefile b/ipu/Makefile
index 5b9ab74..97241e3 100644
--- a/ipu/Makefile
+++ b/ipu/Makefile
@@ -21,6 +21,7 @@ install:
 	cp -P $(LIBNAME).* $(DEST_DIR)/usr/lib
 	@mkdir -p $(DEST_DIR)/usr/include
 	cp mxc_ipu_hl_lib.h $(DEST_DIR)/usr/include
+	cp ipu.h $(DEST_DIR)/usr/include
 else
 all install:
 	@echo "Not support platform, will not complile"
-- 
1.8.0

